vue-components/stories/BouncingDots.stories.ts

```diff
@@ -0,0 +1,39 @@
+import BouncingDots from '@/components/BouncingDots';
+import { bouncingDotsSizes } from '@/components/bouncingDots';
+import { Component } from 'vue';
+
+export default {
+    component: BouncingDots,
+    title: '/Loader/BouncingDots',
+};
+
+export function basic( args: { content: string; underlined: boolean } ): Component {
+    return {
+        data(): object {
+            return { args, bouncingDotsSizes };
+        },
+        components: { BouncingDots },
+        props: Object.keys( args ),
+        template: `
+			<div>
+				<div v-for="size in bouncingDotsSizes">
+                    {{ size }}<br><br>
+                    <BouncingDots :size="size"></BouncingDots><br><br>
+				</div>
+            </div>
+		`,
+    };
+}
+
+basic.args = {
+    size: 'small',
```

Comment:
I think that using a selector control for size is the best choice. Like that, the options are visible for users.